File: D7SMS-AngularJs/d7smslib/Configuration.js
/**
 * D7SMSLib
 *
 */

;(function (angular) {
'use strict';

angular.module('D7SMSLib')
    .factory('Configuration', [Configuration]);

    function Configuration() {
        return {
            //The base Uri for API calls
            BASEURI : 'https://rest-api.d7networks.com/secure',

            //API Key
            aPIUsername : '',

            //API Token
            aPIPassword : ''

        };
    }

}(angular));

File: D7SMS-Ruby/lib/d7_sms/configuration.rb
# d7_sms
#
# https://d7networks.com ).

# CohesityManagementSdk
module D7Sms
  # All configuration including auth info and base URI for the API access
  # are configured in this class.
  class Configuration
    # The base Uri for API calls
    @base_uri = 'https://rest-api.d7networks.com/secure'

    # API Key
    @api_username = 'TODO: Replace'

    # API Token
    @api_password = 'TODO: Replace'

    # The attribute accessors for public properties.
    class << self
      attr_accessor :array_serialization
      attr_accessor :base_uri
      attr_accessor :api_username
      attr_accessor :api_password
    end
  end
end
